### PR TITLE
[bitnami/postgresql-ha] add loadBalancingOnWrite behavior configuration property

### PR DESCRIPTION
**Pull request by dani8art**
**Description of the change**

Add a new property to configure the load balancing behavior. We are able to do it since https://github.com/bitnami/bitnami-docker-pgpool/pull/17

**Benefits**

Users can select this behavior

**Possible drawbacks**

N/A

**Applicable issues**

  - fixes #

**Additional information**

N/A

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [x] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

### CODE REVIEW

File: bitnami/postgresql-ha/values-production.yaml [hunk 1]
```diff
@@ -236,15 +236,13 @@ postgresql:
   ## ref: https://github.com/bitnami/bitnami-docker-postgresql-repmgr#configuration-file
   ##
   # repmgrConfiguration: |-
-
```
Comment (miguelaeh) [inline bitnami/postgresql-ha/values-production.yaml:239]: Do not remove the empty lines

File: bitnami/postgresql-ha/templates/pgpool/deployment.yaml [hunk 2]
```diff
@@ -65,6 +65,10 @@ spec:
             {{- end }}
             - name: PGPOOL_ENABLE_LOAD_BALANCING
               value: {{ ternary "yes" "no" .Values.pgpool.useLoadBalancing | quote }}
+            {{- if .Values.pgpool.useLoadBalancing }}
+            - name: PGPOOL_DISABLE_LOAD_BALANCE_ON_WRITE
```
Comment (miguelaeh) [inline bitnami/postgresql-ha/templates/pgpool/deployment.yaml:69]: The env var is called `PGPOOL_DISABLE_LOAD_BALANCE_ON_WRITE` and the parameter `pgpool.disableLoadBalancingOnWrite`, that seems to be contradictory

File: bitnami/postgresql-ha/README.md [hunk 3]
```diff
@@ -135,6 +135,7 @@ The following table lists the configurable parameters of the PostgreSQL HA chart
...
 | `pgpool.configurationCM`                       | ConfigMap with the Pgpool configuration file (Note: Overrides `pgpol.configuration`)                                                                                 | `nil` (The value is evaluated as a template)                 |
 | `pgpool.useLoadBalancing`                      | If true, use Pgpool Load-Balancing                                                                                                                                   | `true`                                                        |
+| `pgpool.loadBalancingOnWrite`                      | Specify load balance behavior after write queries appear ('off', 'transaction', 'trans_transaction', 'always'). Load-Balancing                                                                                                                                   | `transaction`                                                        |
```
Comment (miguelaeh) [inline bitnami/postgresql-ha/README.md:138]: `balance` -> `balancer`

File: bitnami/postgresql-ha/values-production.yaml (hunk 1 above)
Comment (dani8art) [reply bitnami/postgresql-ha/values-production.yaml:239]: it was done by my linter automatically, fixing it!

File: bitnami/postgresql-ha/templates/pgpool/deployment.yaml (hunk 2 above)
Comment (dani8art) [reply bitnami/postgresql-ha/templates/pgpool/deployment.yaml:69]: Yes, your right! changing it!

**Comment by miguelaeh** [comment]
Wait for the CI/CD to update the tags

**Review by carrodher — APPROVED** [review]
Are some changes required in the Docker image to support the new env. variable? If so, LGTM if the image was already released and included in the current chart version, in another case we should ensure the image tag is updated in the PR

**Comment by nc-ruth** [comment]
Hi @dani8art @carrodher @miguelaeh, what's the status of this PR?

**Comment by javsalgar** [comment]
It seems that it got lost. Closing this one and creating a new one

https://github.com/bitnami/charts/pull/7134

**Pull request closed without merging**